.pre-commit-config.yaml

```diff
@@ -9,32 +9,36 @@ exclude: |
 
 repos:
   - repo: https://github.com/pre-commit/mirrors-clang-format
-    rev: v17.0.6
+    rev: v20.1.0
     hooks:
       - id: clang-format
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.4.4
+    rev: v0.11.2
```

Comment:
Same here, `master` has `0.9.10`

I'm not going to point them all out, but ones are in a similar situation with slightly newer versions, and I assume the reason behind them all is the same